pkg/activator/handler/handler.go

```diff
@@ -226,8 +227,16 @@ func (a *activationHandler) proxyRequest(w http.ResponseWriter, r *http.Request,
 	network.RewriteHostIn(r)
 	recorder := pkghttp.NewResponseRecorder(w, http.StatusOK)
 	proxy := httputil.NewSingleHostReverseProxy(target)
-	proxy.Transport = &ochttp.Transport{
-		Base: a.transport,
+	config := activatorconfig.FromContext(r.Context())
+	isTraceEnabled := false
+	if config != nil {
+		isTraceEnabled = config.Tracing.Enable
+	}
+	proxy.Transport = a.transport
```

Comment:
Does the probeTransport need the same treatment?

Reply:
Technically, yes, but probes are about to become decoupled from the requests themselves so I don't think it matters much.